schemas/documents/registry/buyersPage.js

```diff
@@ -0,0 +1,45 @@
+export default {
+  name: 'buyersPage',
+  type: 'document',
+  title: 'For Buyers Page',
+  __experimental_actions: ['update', 'create', 'delete', 'publish'],
+  fields: [
+    {
+      name: 'heroSection',
+      type: 'heroSection',
+      title: 'Hero Section',
+      validation: Rule => Rule.required(),
+    },
+    {
+      name: 'imageGridSection',
+      type: 'imageGridSection',
+      title: 'Image Grid Section',
+      validation: Rule => Rule.required(),
+    },
+    {
+      name: 'featuredSection',
+      type: 'reference',
+      to: [{type: 'featuredSection'}],
+      title: 'Featured Credit Section',
+      validation: Rule => Rule.required(),
+    },
+    {
+      name: 'faqSection',
+      type: 'bottomBanner',
+      title: 'FAQ Section',
+      validation: Rule => Rule.required(),
+    },
+    {
+      name: 'footerButtonText',
+      type: 'string',
+      title: 'Footer Button Text',
+      validation: Rule => Rule.required(),
+      default: '',
```

Comment:
why using a default here?

Reply:
removed. I was experimenting with using defaults, so it would clean up the react code a bit. Without it, we have to give each field a fall back like `content?.buttonText || ''` which feels cumbersome. But I'm not sure if it is better to have a default. I'll just leave it out.

Reply:
but in this particular case, wouldn't we just not display the button at all? (ie `{content?.buttonText && <Button>...}`